src/android/AndroidManifest.XR.xml

```diff
@@ -9,19 +9,35 @@
   <uses-feature android:name="android.hardware.vr.high_performance" android:required="true" />
   <uses-feature android:name="android.software.vr.mode" android:required="true" />
 
+  <!-- Needed for Oculus devices -->
+    <uses-feature android:name="com.oculus.feature.PASSTHROUGH" android:required="false" />
+    <meta-data android:name="com.oculus.intent.category.VR" android:value="vr_only"/>
+    <meta-data android:name="com.oculus.supportedDevices" android:value="quest|quest2|cambria"/>
+  <!-- -->
+
   <application
       android:allowBackup="true"
       android:icon="@android:drawable/ic_info"
       android:label="${appLabel}"
       android:theme="@style/Theme.AppCompat.NoActionBar"
       tools:targetApi="31">
+    <!-- Needed for Oculus devices -->
+      <uses-native-library
+          android:name="libopenxr_forwardloader.oculus.so"
```

Comment:
What happens if we forget to include this on an oculus device? Do we have a clear error with some undefined symbol or something handled by the XR runtime, or is it crashing in a weird way?